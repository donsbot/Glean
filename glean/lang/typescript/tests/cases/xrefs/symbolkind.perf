{ "@generated": null, "scip.SymbolKind.1": 14 }