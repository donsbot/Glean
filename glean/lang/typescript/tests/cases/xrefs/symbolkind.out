[
  "@generated",
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/"
      },
      "kind": 3
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/Git#"
      },
      "kind": 4
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/Git#`<constructor>`()."
      },
      "kind": 5
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/Git#`<constructor>`().(dir)"
      },
      "kind": 7
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/Git#commit()."
      },
      "kind": 5
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/Git#commit().(author)"
      },
      "kind": 7
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/Git#commit().(date)"
      },
      "kind": 7
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/Git#commit().(msg)"
      },
      "kind": 7
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/createTempRepo()."
      },
      "kind": 5
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/createTempRepo().typeLiteral10:git."
      },
      "kind": 12
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm . . home/dons/Glean/glean/lang/typescript/tests/cases/xrefs/`example.ts`/createTempRepo().typeLiteral10:repoDir."
      },
      "kind": 12
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm typescript 4.9.5 lib/`lib.es2022.error.d.ts`/Error#"
      },
      "kind": 4
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm typescript 4.9.5 lib/`lib.es5.d.ts`/Error#"
      },
      "kind": 4
    }
  },
  {
    "key": {
      "symbol": {
        "key": "scip-typescript npm typescript 4.9.5 lib/`lib.es5.d.ts`/Error."
      },
      "kind": 12
    }
  }
]